packages/next/client/image.tsx

```diff
@@ -425,9 +438,14 @@ export default function Image({
         `Image with src "${src}" has both "priority" and "loading='lazy'" properties. Only one should be used.`
       )
     }
-    if (sizes && layout !== 'fill' && layout !== 'responsive') {
+    if (
+      sizes &&
+      layout !== 'fill' &&
+      layout !== 'responsive' &&
+      layout !== 'raw'
+    ) {
       console.warn(
-        `Image with src "${src}" has "sizes" property but it will be ignored. Only use "sizes" with "layout='fill'" or "layout='responsive'".`
+        `Image with src "${src}" has "sizes" property but it will be ignored. Only use "sizes" with 'fill,' 'responsive,' or 'raw' layout modes.`
```

Comment:
```suggestion
        `Image with src "${src}" has "sizes" property but it will be ignored. Only use "sizes" with "layout='fill'", "layout='responsive'", or "layout='raw'".`
```

Lets keep this explicit since the warning could show up if you omit `layout` and it wouldn't be clear that its a prop

Reply:
Sure